templates/github/.github/workflows/update-labels.yml.j2

```diff
@@ -0,0 +1,31 @@
+{% include 'header.j2' %}
+{% from 'macros.j2' import
+  checkout,
```

Comment:
Now we are importing and not using it. One of these conditions should change.

Reply:
Fixed.